tests/test_cpp_running_simulations.py

```diff
@@ -35,14 +35,14 @@ def test_cpp_running_simulations():
     example_number = 1
 
     # input and output filenames (these must have the .h5 extension)
-    input_filename  = 'example_input.h5'
+    input_filename = 'example'
```

Comment:
why did you separate the suffix?

Reply:
You're right. this doesn't make sense...